### DISCUSSION TITLE
How to stop QuickNote from starting at the bottom?

### DISCUSSION THREAD

**Discussion by PrSunflower** [gsantner/markor#1696] [Q&A] [answered]
Hello fellow community, I hope you're well 🙂 

Since some versions when I open Markor > QuickNote it immediately scrolls to the bottom of the document.

Previously it started at **the top** of the document and I prefer that way.

Is there a way to configure this?

**Comment by harshad1** [comment, marked as answer]
You mean the quicknote tab? That will always start at the bottom.

The quicknote file itself (launched through the file manager, a shortcut or the widget) will start at the last seen position.

**Reply by harshad1** [reply]
Note that you can grab the scrollbar and drag if you want to get to the top fast

**Reply by PrSunflower** [reply]
Hello, thank you for your answer. Yes the quicknote tab. So there isn't any setting to make it start at the top?

I'm asking because there is an option for all documents, Settings > Edit mode > "Start on bottom" which I have deactivated.

**Reply by PrSunflower** [reply]
> Note that you can grab the scrollbar and drag if you want to get to the top fast

Oh yeah I know about the scroll bar but still it's annoying when I write stuff 5 times a day 😅

**Comment by harshad1** [comment]
Start on bottom forces all documents to start at the bottom. With this disabled, it will start at the last known position. I have a PR in progress which relaxes the start on bottom conditions slightly.





---- On Thu, 26 May 2022 11:05:20 -0700 Pr. Sunflower ***@***.***> wrote ----





Hello, thank you for your answer. Yes the quicknote tab. So there isn't any setting to make it start at the top?

I'm asking because there is an option for all documents, Settings > Edit mode > "Start on bottom" which I have deactivated.

**Reply by PrSunflower** [reply]
Thank you for your answers 🙂
